### ISSUE TITLE
Bug: test set contains all graph nodes in epgm-example.py

### ISSUE THREAD

**Issue by youph** [stellargraph/stellargraph#104]
### Description
/stellar-ml/demos/node-classification/epgm-example.py, line 189: 
```
test_ids = list(G.nodes())
```
This means the test set includes both the training and validation sets as well.
Should be:
```
test_ids = [v[0] for v in test_nodes]
```
My guess is this explains the discrepancy between validation and test loss and accuracies, that I mentioned earlier.

### Done Checklist (Development)
- [x] Error fixed
- [x] Branch and Pull Request build on CI
- [x] Branch and Pull Request pass unit tests on CI
- [x] Branch and Pull Request pass integration tests on CI
- [x] Peer Code Review Performed
